dora/core/server/master/src/main/java/alluxio/master/job/CopyJob.java

```diff
@@ -609,11 +630,14 @@ public CopyProgressReport(CopyJob job, boolean verbose)
       }
       mFailureReason = job.mFailedReason.orElse(null);
       mFailedFileCount = job.mFailedFiles.size();
+      mSkippedFileCount = job.mCurrentSkipCount.get();
       if (verbose && mFailedFileCount > 0) {
         mFailedFilesWithReasons = job.mFailedFiles;
       } else {
         mFailedFilesWithReasons = Collections.emptyMap();
       }
+      mStartTime = job.mStartTime;
+      mEndTime = job.mEndTime.orElse(0);
```

Comment:
I'm not sure orElse 0 is a good idea here since we should raise an exception if we get endtime when unfinished